index.bs

```diff
@@ -325,8 +319,20 @@ interface PictureInPictureWindow {
 };
 </pre>
 
-The {{PictureInPictureWindow}} is a simple proxy to get information such as
-dimensions about the current Picture-in-Picture window.
+A {{PictureInPictureWindow}} instance represents a <a>Picture-in-Picture
+window</a> associated with an {{Element}}. When instantiated, an instance of
```

Comment:
Could it be `{{HTMLVideoElement}}` instead of `{Element}`? I mean, the spec today only mention videos, right?

Reply:
Done. Thanks!